storage/remote/read.go

```diff
@@ -219,6 +199,40 @@ func (q externalLabelsQuerier) addExternalLabels(ms []*labels.Matcher) ([]*label
 	return ms, el
 }
 
+// LabelValues implements storage.Querier and is a noop.
```

Comment:
I can't find it right now but I feel like I've seen an issue or WIP PR about this.

Reply:
ack will search and link